libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -547,8 +547,8 @@ void AP_SerialManager::init()
                     break;
 #endif
                     
-                case SerialProtocol_EFI_MS:
-                    state[i].baud = AP_SERIALMANAGER_EFI_MS_BAUD;   // update baud param in case user looks at it
+                case SerialProtocol_EFI:
+                    state[i].baud.set_default(AP_SERIALMANAGER_EFI_MS_BAUD);
```

Comment:
Still referencing `_MS` here.  I guess it's the default for that particular device, though....

Reply:
indeed